docs_user/modules/openstack-keystone_adoption.adoc

```diff
@@ -79,7 +79,7 @@ control plane (everything except Keystone service and endpoints):
 ----
 openstack endpoint list | grep keystone | awk '/admin/{ print $2; }' | xargs ${BASH_ALIASES[openstack]} endpoint delete || true
 
-for service in aodh cinderv3 glance manila manilav2 neutron nova placement swift; do
+for service in aodh cinderv3 glance gnocchi manila manilav2 neutron nova placement swift; do
```

Comment:
Can you please copy the list into the tests? I see it already got a bit out of sync, but thanks to the `|| true` after `service delete`, any additions shouldn't fail the CI, even if the service didn't exist in the Standalone source deployment. Here is the list in the tests:

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/f31fa894302f856d55b12c2f010ca15d32660069/tests/roles/keystone_adoption/tasks/main.yaml#L87

Reply:
Done